tests/Hydra/Serializer/DocumentationNormalizerTest.php

```diff
@@ -168,6 +169,21 @@ public function testNormalize()
                             'hydra:writable' => true,
                             'hydra:description' => 'description',
                         ],
+                        [
+                            '@type' => 'hydra:SupportedProperty',
+                            'hydra:property' => [
+                                '@id' => '#dummy/jsonData',
+                                '@type' => 'rdf:Property',
+                                'rdfs:label' => 'jsonData',
+                                'domain' => '#dummy',
+                                'range' => 'xmls:list',
```

Comment:
According to the spec, it doesn't look ok: `·List· datatypes are always ·constructed· from some other type; they are never ·primitive·.`.

If we need to go this way, we'll need to provide our XSD with custom types, and use those custom types here.

Reply:
Providing our XSD with custom types doesn't seem a good idea to me, for interoperability for example.
WDYT? Do you have another solution in mind?

Reply:
XML and open data in general doesn't fit well with random, dynamic data structures... The best thing to do is that the user document correctly the expected structure of the returned data (for example using https://github.com/dunglas/doctrine-json-odm.

Reply:
Sure but it can also be a simple array of strings or integers. Doctrine JSON ODM does not manage this kind of JSON, doesn't it?